File: dbflow-core/src/main/java/com/raizlabs/android/dbflow/annotation/MultiCacheField.java
package com.raizlabs.android.dbflow.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Description: Marks a field as the IMultiKeyCacheModel that we use to convert multiple fields into
 * a single key for caching.
 */
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.SOURCE)
public @interface MultiCacheField {
}

File: dbflow-core-common/src/main/kotlin/com/raizlabs/dbflow5/annotation/MultiCacheField.kt
package com.raizlabs.dbflow5.annotation

/**
 * Description: Marks a field as the IMultiKeyCacheModel that we use to convert multiple fields into
 * a single key for caching.
 */
@Target(AnnotationTarget.FIELD)
@Retention(AnnotationRetention.SOURCE)
annotation class MultiCacheField
